### PR TITLE
Feat: refresh github wrapped to 2023

### PR DESCRIPTION
**Pull request by TheJoin95**
Changelog
- update UI
- improve social buttons
- improve social image generate via netlify edge functions
- update github graph api query
- add a new card

image

### CODE REVIEW

File: src/components/Cards/RecapNumTxtCard.tsx [hunk 1]
```diff
@@ -9,17 +9,17 @@ const RecapNumTxtCard = ({ nCommits, txt }: { nCommits: number, txt: string }) =
...
                     <rect width="280" height="60.6667" transform="translate(40 404.667)" fill="#000100" />
-                    <text textAnchor="middle" fill="#B75F0D" xmlSpace="preserve" style={{whiteSpace: 'pre'}} fontFamily="Helvetica" fontSize="80.8889" fontWeight="500" letterSpacing="-0.02em"><tspan x="180.1111" y="467.337">{nCommits}</tspan></text>
+                    <text textAnchor="middle" fill="#B75F0D" xmlSpace="preserve" style={{whiteSpace: 'pre'}} fontFamily="Helvetica" fontSize="80.8889" fontWeight="500" letterSpacing="-0.02em"><tspan x="180.1111" y="467.337">{number}</tspan></text>
                 </g>
-                <text fill="#000100" xmlSpace="preserve" style={{whiteSpace: 'pre'}} alignmentBaseline="central" fontFamily="Helvetica" fontSize="50" fontWeight="500" letterSpacing="-0.02em"><tspan x="80.614" y="650.704">{txt}</tspan></text>
+                <text fill="#000100" xmlSpace="preserve" style={{whiteSpace: 'pre'}} alignmentBaseline="central" fontFamily="Helvetica" fontSize={txt.length > 15 ? "37" : "50"} fontWeight="500" letterSpacing="-0.02em"><tspan x={txt === 'Commits' ? "80.614" : "35"} y="650.704">{txt}</tspan></text>
```
Comment (TheJoin95) [inline src/components/Cards/RecapNumTxtCard.tsx:22]: Note for reviewer: doing this for future implementation with Google/Microsoft

File: src/components/Cards/RecapTopCard.tsx [hunk 2]
```diff
@@ -25,22 +26,21 @@ const RecapTopCard = (githubInfos: TGitHubUser, img: any) => {
...
                 </g>
-                <text textAnchor="middle" fill="#e2e441" xmlSpace="preserve" style={{whiteSpace: 'pre'}} alignmentBaseline="central" fontFamily="Helvetica" fontSize="35" fontWeight="500" letterSpacing="-0.02em"><tspan x="180.614" y="480.704">{usernamePadded}</tspan></text>
                 <g clipPath="url(#clip3_10_1203)">
-                    <text fill="#fff" xmlSpace="preserve" fontFamily="Helvetica" fontSize="16" fontWeight="500" letterSpacing="0em"><tspan x="32.6562" y="520.991">Top Repos and 🌟</tspan></text>
-                    {githubInfos?.user.repositories?.edges.map(
+                    <text fill="#fff" xmlSpace="preserve" alignmentBaseline="central" fontFamily="Helvetica" fontSize="16" fontWeight="500" letterSpacing="0em"><tspan x="32.6562" y="520.991">Most PR reviewed</tspan></text>
+                    {top5reviewContributions.map(
```
Comment (TheJoin95) [inline src/components/Cards/RecapTopCard.tsx:31]: Note for reviewer: removing the fancy mechanism of stars in favour of top contributions

File: src/components/layout.tsx [hunk 3]
```diff
@@ -11,41 +12,36 @@ export default function Layout({ children }: any) {
...
-          property="og:description"
-          content="Github Wrapped 2023 - By Schrodinger Hat is the wrapped of the year for your open source contributions"
-        />
-        <meta
-          property="og:image"
-          content={`https://wrapped.schrodinger-hat.it/.netlify/functions/generator?username=${user}`}
-        />
 
+        <meta property="og:title" content={"Github Wrapped 2023 - By Schrodinger Hat"} />
+        <meta property="og:type" content="article" />
```
Comment (TheJoin95) [inline src/components/layout.tsx:17]: Did this as we did for osday (which is working on linkedin, facebook and twitter)

**Comment by nicpuppa** [comment]
Hey @TheJoin95, is this PR ready to be reviewed ?

**Comment by TheJoin95** [comment]
@nicpuppa You can review all expect for the netlify folder

**Comment by nicpuppa** [comment]
@TheJoin95 the sign in with github seems broken 

Screenshot 2023-12-12 at 09 14 17

**Comment by TheJoin95** [comment]
@nicpuppa could you please test it out? Check also the og:image

**Review by nicpuppa — APPROVED** [review]

**Pull request merged**